frontend/src/features/properties/selector/components/MapClickMonitor.tsx

```diff
@@ -46,7 +46,7 @@ export const mapFeatureToProperty = (
     longitude: selectedFeature?.properties?.CLICK_LAT_LNG?.lng ?? latLng.lng ?? undefined,
     planNumber: selectedFeature?.properties?.PLAN_NUMBER ?? undefined,
     address: 'placeholder', //todo: need alternate source for this
-    legalDescription: 'placeholder', //todo: need access to fully attributed parcelmap bc layer,
+    legalDescription: selectedFeature?.properties?.LEGAL_DESCRIPTION,
```

Comment:
Ah I see why you changed the parcel map URL earlier. We do need FA parcel map to get legal description

Reply:
Probably relevant to alert testers of the change in map layer so they run regression against everything that uses parcel map layer queries...